src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -194,6 +193,7 @@ public TopicPartitionChannel(
         topicPartition,
         channelName,
         tableName,
+        false,
```

Comment:
add a single line comment here saying what this false represents

Reply:
Added.